NEWS.md

```diff
@@ -79,6 +79,12 @@ function calls. (#850, #851, @renkun-ken)
 * Debugging functions (`browser()`, `debug()`, `debugcall()`, `debugonce()`, `trace()`, `undebug()`, `untrace()`) are now part of the default set of undesirable functions to help prevent them from being committed by mistake. (#876, @michaelchirico)
 * New linter `package_hooks_linter()` runs a series of checks also done by `R CMD check` on the `.onLoad()`, `.onAttach()`, `.Last.lib()` and `.onDetach()` hooks (#882, @MichaelChirico)
 * Improved location information for R parse errors (#894, #892, @renkun-ken and @AshesITR)
+* New tag based documentation pages for linters (#888, @AshesITR)
```

Comment:
it would help to motivate this (why as lintr user do I care about this update?). maybe a quick blurb like

> Add new tag-based documentation pages for linters to help organize information surrounding the growing number of linters available

Reply:
Refining:

> To help organize information surrounding the growing number of linters available, we've added metadata "tags" for each linter `(#888)`.

> bullet: each tag gets a help page, e.g. ?x

WDYT